website/docs/d/public_ip_prefix.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_public_ip_prefix"
+sidebar_current: "docs-azurerm-datasource-public-ip-prefix-x"
+description: |-
+  Gets information about an existing Public IP Prefix.
+
+---
+
+# Data Source: azurerm_public_ip_prefix
```

Comment:
could we add an entry for this new data source [to the sidebar](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L57-L59)?